Makefile

```diff
@@ -1,14 +1,21 @@
-lint:
-	if [ ! -f ./bin/golangci-lint ] ; \
-	then \
-		curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.32.2; \
-	fi;
-	./bin/golangci-lint run -e gosec
+.PHONY: test docs docker-image
 
 test:
-	go test -covermode=count -coverprofile=count.out -v ./...
+	go test -covermode=count -coverprofile=count.out -v $(go list ./... | grep -v /docs-test/)
+
+	export PATH=$$PATH:`go env GOPATH`/bin; make -C rest-api test-doc
```

Comment:
@kairoaraujo It looks like now that the `test` recipe eventually leads to a call to `swag`, the GitHub workflow is broken since the ubuntu-latest container shell environment can't find the `swag` command. See: https://github.com/kairoaraujo/goca/actions/runs/3142411740/jobs/5105977765

Which is confusing because it looks like the `check_swag` recipe which we depend on should handle this.
Checking the branch out and running the tests locally on my machine does indeed auto-magically download `swag`.
Perhaps it is something to do with how `$PATH` is handled in the ubuntu-latest container?

Reply:
Thank you @necheffa, I will find time to fix it this weekend :)